packages/config/src/options.ts

```diff
@@ -169,6 +169,11 @@ const driverConfigOptions: Array<DriverConfigOption> = [
       indexHtmlFile: 'cypress/support/component-index.html',
     },
     validation: isValidConfig,
+  }, {
+    name: 'defaultBrowser',
+    defaultValue: null,
+    validation: validate.isString,
```

Comment:
We should probably restart the browser if the user changed the config value while Cypress is open. Probably want to verify this works correctly as well.
```suggestion
    validation: validate.isString,
    requireRestartOnChange: 'browser',
```